static/app/views/releases/list/releaseCard/releaseCardProjectRow.tsx

```diff
@@ -208,6 +236,38 @@ function ReleaseCardProjectRow({
           </Tooltip>
         </NewIssuesColumn>
 
+        {hasThresholds && (
+          <DisplaySmallCol>
+            {/* TODO: link to release details page */}
+            {thresholds && thresholds.length > 0 && (
+              <Tooltip
+                title={
+                  <div>
+                    <div>
+                      {`${healthyThresholds.length} / ${thresholds.length} ` +
+                        t('thresholds succeeded')}
+                    </div>
+                    {pendingThresholds.length > 0 && (
+                      <div>
+                        {`${pendingThresholds.length} / ${thresholds.length} ` +
+                          t('still pending')}
+                      </div>
+                    )}
+                    {t('Open in Release Details')}
```

Comment:
will this be a link?

Reply:
This is just a popover - but the number it wraps will eventually be a link - i just need to build out the details page view which i'm doing now